000_example/español/AnaColon/resumen de Lesser MP

```diff
@@ -0,0 +1,25 @@
+
+Muchos arrecifes de coral están en ambientes de su límite superior de temperatura.
+
+El factor independiente fue el cambio de temperatura que fue hecha en el experimento al cambiar la velocidad de flujo, y consecuentemente el tiempo de residencia, del agua de mar en los tanques aguantando las muestras de coral. La temperatura de lo corales se mantuvo entre 2 y 3 grados Celsius más alto de lo que se registró en el pedazo de arrecife. Otro factor que fue medido por separado fue coral que estaban en temperaturas elevada y en la presencia de exógenos antioxidantes ascorbato y catalasa. 
+
+Los investigadores midieron la fotosíntesis de ciertas colonias en varias irradiancias y evaluó el número de zooxantelas perdidas en el agua de mal y el modo de exclusión producido de los diferentes tratamientos.  
+
+Los investigadores midieron el total de zooxantelas a las 3 de la tarde y notaron que la palidez visible fue visto a las 2 de la tarde para coral que paso por blanqueamiento. Esto significa que todas las medidas en corales pasando por blanqueamiento fueron hecho una hora después de la ocurrencia del blanqueamiento visible. 
+
+Corales en la presencia de exógenos antioxidantes que fueron expuesto a temperaturas elevadas no enseñaron cambios distinguibles en la velocidad de fotosíntesis y respiración. 
+
+Corales en la presencia de antioxidantes con temperaturas elevadas no enseñaron señales de muerte de tejido huésped. Corales expuestos a temperaturas elevadas no tuvieron blanqueamiento total. 
+
+Los corales con ascorbato y catalasa tuvieron un porciento de zooxantelas en células gastrodérmicas liberadas del huésped que era menos de los corales que fueron a temperaturas elevadas solamente. Resultados similares fueron visto entre manitol antioxidantes y corales sin antioxidantes. 
+
+El número de zooxantelas perdido por unidad de área fue más en los corales expuestos a temperaturas elevadas comparado a los corales expuesto a temperaturas elevadas y antioxidantes. 
+
+Zooxantelas cultivadas que tuvieron la adición de carroñeros exógenos de radicales de oxígeno enseñaron una restauración visible en una porción significante en las capacidades de fotosíntesis. 
+
+Un bajo porcentaje de células gastrodérmicas con zooxantelas en el estudio fue encontrado a tener consistencia con exocitosis como el mecanismo principal de la perdida de zooxantelas. 
+
+La evidencia presentada en este estudio encontró que el blanqueamiento de arrecifes de coral debido a temperaturas elevadas puede ser prevenido con antioxidantes. Esto apoya a un papel para estrés oxidativo como un mecanismo subyacente para el blanqueamiento inducido por temperatura. 
```

Comment:
if rewriting for a blog, you may want to include a little bit of modern context with claims like these (in the following 25 years, have these findings been supported? refuted? never again tested? are there caveats that we've thought about since then, like practicability (a solution for bleaching in a tank might not be possible on the scale of reefs or ecosystems?)